configuration/arm64/docker-compose.yml

```diff
@@ -16,14 +14,15 @@ services:
       - mysql_root_password
     volumes:
       - ./mysql/localhost.sql:/docker-entrypoint-initdb.d/localhost.sql
-      - ./mysql/port_drayage.sql:/docker-entrypoint-initdb.d/port_drayage.sql
+      # - ./mysql/port_drayage.sql:/docker-entrypoint-initdb.d/port_drayage.sql
```

Comment:
Is the idea here that we do not want to default to having port drayage functionality on?

Reply:
@paulbourelly999 Yeah. It's not used by anybody at the moment. It'll be better to have it commented out unless somebody will use it.

Reply:
Leave a comment indicating how to get port drayage functionality back and/or to come up with a more elegant solution to not including port drayage by default.

Reply:
All, I think we need to leave the port_drayage as is. The expectation is that whatever was deployed in the past should be backwards compatible or scenarios would continue to work as is, unless we notify the client we're removing this feature or retiring it.

Reply:
Changed back